### PR TITLE
小的改动

### PR DESCRIPTION
**Pull request by liuditai**

### CODE REVIEW

File: basic/views/test-excution/my-missions.php [hunk 1]
```diff
@@ -32,12 +32,21 @@
                     return $repoLinks;
                 }
             ],
+            [
+                'attribute' => 'state',
+                'content' => function ($model, $key, $index, $column) {
+                    if ($model['state'] === 'c') {
+                        return 'complated';
```
Comment (yangzhenping) [inline basic/views/test-excution/my-missions.php]: typo, completed, thanks.

**Review by yangzhenping — CHANGES_REQUESTED** [review]
2 typos, please correct them, thanks.

**Comment by liuditai** [comment]
拼写错误修改完成

**Review by yangzhenping — APPROVED** [review]
thank you!

**Pull request merged**